Airports/EGLF/VRPs.txt

```diff
@@ -1,4 +1,11 @@
+*M3 Jct. 3:N051.21.35.000:W000.40.85.000
 *Bagshot:N051.20.57.012:W000.41.57.011
-*Nokia Factory:N051.17.32.855:W000.47.53.267
+*M3 Jct. 4:N051.18.98.333:W000.45.41.667
+*Fleet Pond:N051.17.31.667:W000.49.48.333
+*Hook:N051.16.76.667:W000.57.71.667
 *Guildford:N051.14.21.984:W000.35.05.999
-*Alton:N051.09.06.984:W000.57.57.995
\ No newline at end of file
+*Tongham:N051.13.70.000:W000.44.53.333
+*Farnham Castle:N051.13.11.667:W000.48.13.333
+*Frensham Great Pond:N051.09.26.667:W000.47.50.000
+*Alton:N051.09.06.984:W000.57.57.995
+;*Nokia FactoryN051.17.32.855:W000.47.53.267
```

Comment:
I'd just remove rather than comment out - no indication that it will be coming back :)